Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m

```diff
@@ -33,6 +38,13 @@ - (instancetype)init
     return [[SEGTaplyticsIntegration alloc] initWithSettings:settings];
 }
 
+- (id<SEGIntegration>)createWithoutSettings
+{
+    return [[SEGTaplyticsIntegration alloc] init];
```

Comment:
where is the `init` method defined? your new initializer was called `initWithoutSettings: andTaplytics:(id)taplyticsClass;`.